estrategiacontenidos/mapadelsitio.md

```diff
@@ -1,16 +1,21 @@
 # Mapa del sitio
 
-<!--
-Recoged el mapa del sitio en este documento en forma de lista
-formateada usando la sintaxis de Markdown, así:
-
-- si queréis
-- una lista
-  + sin ordenar
-
-o así:
-
-1. si queréis
-   1. una lista
-   2. ordenada
--->
+1. Página principal
+    1. **Podcast**
+         1. Noticiario 1
+             - Audio noticiario 1
+         2. Noticiario 2
+             - Audio noticiario 2
+         3. Noticiario 3
+              - Audio noticiario 3
```

Comment:
Entiendo que tendríais una sección por cada capítulo del podcast? O a qué corresponden estos "noticiario 1", "noticiario 2" etc?

El audio de cada noticiario, ¿va en la página que habéis titulado "Noticiario #"? ¿O son páginas separadas?

Reply:
Finalmente, hemos decidido poner el audio de los noticiarios en una sola página y eliminar el espacio de tomas falsas porque consideramos que no tendremos suficiente contenido.

Reply:
Es decir, tendréis una sola página "noticiarios" en la que se incluirán los audios de todos lo noticiarios. Si es así, no está correctamente reflejado en el mapa del sitio: si "noticiario 1" no es una página sino un audio incrustado en la página "Podcasts", no debería figurar en el mapa del sitio. El mapa del sitio **solamente** debe mostrar las diferentes páginas que compondrán el sitio y la relación entre ellas (cuáles son sub-páginas de cuáles).

Por otra parte, incluir los audios de todos los noticiarios en una sola página reduce el número de páginas y simplifica la estructura, pero plantea un nuevo problema: no se pueden referenciar (por URL) noticiarios individuales. ¿Podéis prever qué consecuencias tendría esto?

Reply:
Miren, hemos creado una página para cada uno de los audios, es decir, dentro del apartado podcasts, habrá tres apartados con su audio